com.unity.uiwidgets/Runtime/engine/UIWidgetsPanel.cs

```diff
@@ -259,6 +260,11 @@ void CollectGarbageOnDemand()
 #if !UNITY_EDITOR && UNITY_ANDROID
             if (!IsAndroidInitialized()) {return ;}
 #endif
+            if(canvas == null){
```

Comment:
could we add some explanation on why doing this?

Reply:
Add comment?

Reply:
yes, we need a comprehensive why-doing-this here.